docs/dev/development.md

```diff
@@ -1,70 +1,90 @@
 # Local Development Setup
-To use this development environment on OSX, you need to have Docker and Boot2docker installed.
+To use this development environment on OSX, you need to have Java, Docker and VirtualBox installed (Boot2docker is optional).
 
 ## Cloudbreak Deployer
 
-Simplest way to prepare the working environment is to start the Cloudbreak on your local machine is to use the [Cloudbreak Deployer](https://github.com/sequenceiq/cloudbreak-deployer).
+Simplest way to prepare the working environment is to start Cloudbreak on your local machine is to use the [Cloudbreak Deployer](https://github.com/hortonworks/cloudbreak-deployer).
 
-First you need to create a _sandbox_ directory which will store the necessary  configuration files and dependencies of [Cloudbreak Deployer](https://github.com/sequenceiq/cloudbreak-deployer). This directory must be created outside of the cloned Cloudbreak git repository:
+First you need to create a _sandbox_ directory which will store the necessary configuration files and dependencies of [Cloudbreak Deployer](https://github.com/hortonworks/cloudbreak-deployer). This directory must be created outside of the cloned Cloudbreak git repository:
 ```
 mkdir cbd-local
+cd cbd-local
 ```
 
-To start the complete Cloudbreak ecosystem on your machine just execute the following sequence of commands:
+Add the following to the file `Profile` there under cbd-local (IPs could be different based on your network/virtualbox setup):
```

Comment:
A short description about how you can check what is the IP address what you need to specify.